src/main/kotlin/org/partiql/scribe/sql/RexConverter.kt

```diff
@@ -279,7 +328,7 @@ public open class RexConverter(
         val newRexConverter = RexConverter(transform, Locals(relProject.input.type.schema))
         val type = constructor.type as? StructType ?: return null
         if (type.constraints.contains(TupleConstraint.Open(false))
-                .not() || type.constraints.contains(TupleConstraint.Ordered).not()
```

Comment:
Not too sure why the previous un-planning (plan -> AST) for SELECT VALUE to SQL select required the tuple to be ordered. Was giving some issues with the plan with the `Rel` set ops since they were unordered.

Reply:
SELECT VALUE requires orderedness because SQL tuples are ordered. That is, SELECT VALUE is only equivalent to SQL SELECT if the constructor is an ordered tuple.

SQL UNION/INTERSECT/EXCEPT should preserve the ordereness, as schema is known.

Reply:
I see. Thanks for the context. I added back the ordered tuple check in [`b543453` (#56)](https://github.com/partiql/partiql-scribe/pull/56/commits/b5434534a9889ab1849e59d0a555394aeb3b9613).